detox/package.json

```diff
@@ -5,17 +5,17 @@
   "author": "Mattermost, Inc.",
   "devDependencies": {
     "@babel/plugin-transform-modules-commonjs": "7.10.4",
-    "@babel/plugin-transform-runtime": "7.11.0",
-    "@babel/preset-env": "7.11.0",
+    "@babel/plugin-transform-runtime": "7.11.5",
+    "@babel/preset-env": "7.11.5",
     "axios": "0.20.0",
-    "babel-jest": "26.3.0",
+    "babel-jest": "26.5.2",
     "babel-plugin-module-resolver": "4.0.0",
-    "detox": "17.4.4",
-    "jest": "26.4.2",
-    "jest-circus": "26.4.2",
-    "jest-cli": "26.4.2",
+    "detox": "17.8.3",
```

Comment:
this is now outdate.. I'm updating a bunch of dependencies including detox, I can add the flags to the jest runner too.

Reply:
Thanks for the heads up. Yes, please go ahead. There's an issue with the visibility check on iOS 14 but I'll take a look on it next.